cves/CVE-2015-8662.yml

```diff
@@ -294,38 +328,45 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: false
+    note: 
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Vulnerability occurs from a user input that, distrusting this input by
```

Comment:
this sentence is a little hard to read, a word like "by" after "that," would make it easier to read